homeassistant/components/fitbit/sensor.py

```diff
@@ -404,20 +460,23 @@ class FitbitSensorEntityDescription(SensorEntityDescription):
     name="Sleep Start Time",
     icon="mdi:clock",
     scope="sleep",
+    entity_category=EntityCategory.DIAGNOSTIC,
 )
 SLEEP_START_TIME_12HR = FitbitSensorEntityDescription(
     key="sleep/startTime",
     name="Sleep Start Time",
     icon="mdi:clock",
     value_fn=_clock_format_12h,
     scope="sleep",
+    entity_category=EntityCategory.DIAGNOSTIC,
 )
 
 FITBIT_RESOURCE_BATTERY = FitbitSensorEntityDescription(
     key="devices/battery",
     name="Battery",
     icon="mdi:battery",
     scope="settings",
+    entity_category=EntityCategory.DIAGNOSTIC,
 )
```

Comment:
Can we use the device class here?

Reply:
This is planned in an upcoming cleanup pr. The state value for this is a string that my assumption is that it's.not compatible with the device class.

Reply:
That's true!